docs/reference/content/driver/reference/index.md

```diff
@@ -11,7 +11,7 @@ title = "Reference"
 ## Reference
 
 The reference documentation for the Java driver focuses on high-level documentation and use-cases. The Javadoc-generated API reference is
-[here](http://api.mongodb.org/java/3.1/).    
+[here](http://api.mongodb.org/java/3.2/).    
```

Comment:
Done.  The link is a bit funny (http://api.mongodb.org/java/3.2/?), but it works.